punx/tests/test_issue129.py

```diff
@@ -0,0 +1,37 @@
+import h5py
+import numpy as np
+from punx import h5tree
+
+def test_render_muliple_axes_attribute():
+    """Ensure axes attributes are rendered as list of double quoted strings.
+
+    @axes should be saved as an array of byte strings or an array of object
+    strings because the NeXus standard says so. Single strings separated by
+    whitespace or other charachters will not be rendered correctly.
+    """
+    with h5py.File('test.h5', 'w') as f:
+        d = f.create_dataset("bytearray/data", data=[])
+        d.attrs['axes'] = np.array("Words are these things".split(), dtype='S')
+        d = f.create_dataset("stringarray/data", data=[])
+        d.attrs['axes'] = "Words are these things".split()
+        d = f.create_dataset("string/data", data=[])
+        d.attrs['axes'] = "Words are these things"
+
+    report = h5tree.Hdf5TreeView('test.h5').report()
+    print("\n".join(report))
+
+    reference = [
+        'test.h5',
+        '  bytearray',
+        '    data:float64[0] = []',
+        '      @axes = ["Words", "are", "these", "things"]',
+        '  string',
+        '    data:float64[0] = []',
+        '      @axes = ["W", "o", "r", "d", "s", " ", "a", "r", "e", " ", "t",'
+        ' "h", "e", "s", "e", " ", "t", "h", "i", "n", "g", "s"]',
```

Comment:
Not sure how to deal with this situation. I guess it's just wrong that the file contains a byte-string or string instead of an array of strings?

Reply:
The strings is setup by: `d.attrs['axes'] = np.array("Words are these things".split(), dtype='S')`

Is there any difference that can be intercepted within `h5tree`?

Reply:
/zero-term-byte-array/data is converted to <class 'numpy.ndarray'> of <class 'numpy.bytes_'>
/pystring-list/data is converted to <class 'numpy.ndarray'> of <class 'str'>
/pystring/data is converted to <class 'str'> of <class 'str'>

I've tried to clarify the differences between the three possible inputs. How do you want me to handle "str of str" (i.e. a string).